app/lib/pre_assembly/batch.rb

```diff
@@ -98,10 +99,11 @@ def load_checksums(dobj)
 
     # rubocop:disable Metrics/AbcSize
     # rubocop:disable Metrics/MethodLength
-    # @return [boolean] true if all objects succeeded, false if any errors
+    # rubocop:disable Metrics/CyclomaticComplexity
     def process_digital_objects
```

Comment:
I created ticket #957 to simplify this method so we can remove some/all of these rubocop exceptions, but didn't want to do too much in this PR